packages/high-roller-bot/.gitignore

```diff
@@ -1,2 +1,7 @@
-src/store.json
-src/settings.json
\ No newline at end of file
+jest-cache
+jest-coverage/clover.xml
+jest-coverage/coverage-final.json
+jest-coverage/lcov-report/
+jest-coverage/lcov.info
+**/.js
+.env
```

Comment:
I think this is already the default in the root gitignore. If it isnt it should be